sections/event-types.txt

```diff
@@ -1386,6 +1387,7 @@ myDiv.addEventListener("auxclick", function(e) {
 			 |                  | <li>{{MouseEvent}}.{{MouseEvent/buttons}} : value based on all buttons currently     |
 			 |                  |     depressed, <code>0</code> if no buttons pressed</li>                             |
 			 |                  | <li>{{MouseEvent}}.{{MouseEvent/relatedTarget}} : <code>null</code></li>             |
+                         |                  | <li>For {{PointerEvent}} speciftic attributes checkout {{PointerEvent}} spec.</li>   |
```

Comment:
"specific".

Also, the phrasing "For xxx, checkout yyy" reads to me as "check it out, dude". I think this would be more consistent with the existing style if it was "For xxx specific attributes, see the yyy spec."

Reply:
Fair enough. To be honest with you I got the same feeling. But I couldn't remember the exact wording in other places. Fixed now. The {{PointerEvent}} reference still doesn't resolved and gives me a link error in bikeshed. Do you know if I should do something specific here or something is missing from PointerEvent spec.